### ISSUE TITLE
Request for choosing saved places in photo batch management

### ISSUE THREAD

**Issue by fredm13** [xbgmsharp/piwigo-openstreetmap#121]
First, i apologize for my english*
Is it possible to add , in the management of pictures by lot , the possibility choosing a saved place ?
We can choose a saved place photo by photo but not by photo batch management.
Thanks !

**Comment by keinkurt** [comment]
+1 I like have it in the Batch Manager Global Mode and Single Mode

**Comment by CoolMcFly81** [comment]
I really like to have that feature too :-)

**Comment by jheinitz** [comment]
Yes, that would be cool. Makes it easier to tag a lot of photos at once.

**Comment by CoolMcFly81** [comment]
Hope tha will work, at least for the Batch Manager Global Mode. Maybe Single Mode will follow ;-)

**Comment by fredm13** [comment]
Thanks for having taken my request into consideration !

**Comment by keinkurt** [comment]
Thanks @CoolMcFly81, worked like a charm! Hopefully it will be merged soon.
